### PR TITLE
Angularapp

### PR DESCRIPTION
**Pull request by mr-karan**
Angular App to show bear documentation

### CODE REVIEW

**Comment by mr-karan** [comment]
Please review this ASAP :+1: @manoranjanp @tushar-rishav

**Comment by sils** [comment]
this seems the wrong repo to me, this is something new independent from whats here isn't it?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+    <title>coala-bears Documentation</title>
+
+    <script src="https://use.fontawesome.com/663882be62.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext'>
+    <link rel='stylesheet' type='text/css' href='bower_components/materialize/dist/css/materialize.min.css'>
+    <link href="app/styles/main.css" rel="stylesheet">
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
```
Comment (tushar-rishav) [inline index.html:15]: I think this change is temporary? You might wanna switch to cdns at the end?
Just confirming.

Comment (mr-karan) [inline index.html:15]: yes, for production

**Comment by mr-karan** [comment]
moved to the correct place https://gitlab.com/coala/website/merge_requests/2

**Comment by jayvdb** [comment]
And it is now https://gitlab.com/coala/new-website-prototype/merge_requests/2 , and is merged.

The branch `angularapp` can be deleted now?

**Pull request closed without merging**